From: Jan Kara <jack@suse.cz>
Subject: kabi: Add placeholder into struct vm_fault
Patch-mainline: Never, kabi
References: fate#319256

Making cache flush clean dirty bits in the radix tree will need some changes
into fault handling and propagating more information into the fault handlers.
Reserve space in struct vm_fault in case we have to backport those patches.

Signed-off-by: Jan Kara <jack@suse.cz>

---
 include/linux/mm.h |    1 +
 1 file changed, 1 insertion(+)

--- a/include/linux/mm.h
+++ b/include/linux/mm.h
@@ -242,6 +242,7 @@ struct vm_fault {
 	pgoff_t max_pgoff;		/* map pages for offset from pgoff till
 					 * max_pgoff inclusive */
 	pte_t *pte;			/* pte entry associated with ->pgoff */
+	void *suse_kabi_padding;
 };
 
 /*
